shoreline.go

```diff
@@ -142,32 +117,41 @@ func main() {
 
 	httpClient := &http.Client{Transport: tr}
 
-	highwater := highwater.NewHighwaterClientBuilder().
-		WithHostGetter(config.HighwaterConfig.ToHostGetter(hakkenClient)).
-		WithHttpClient(httpClient).
-		WithConfig(&config.HighwaterConfig.HighwaterClientConfig).
-		Build()
-
 	rtr := mux.NewRouter()
 
 	/*
 	 * User-Api setup
 	 */
 
-	var marketoManager marketo.Manager
-	if err := config.User.Marketo.Validate(); err != nil {
-		logger.Println("WARNING: Marketo config is invalid", err)
-	} else {
-		logger.Print("initializing marketo manager")
-		marketoManager, _ = marketo.NewManager(logger, config.User.Marketo)
-		marketoConfig.Set(1)
-	}
-
 	clientStore := user.NewMongoStoreClient(&config.Mongo)
 	defer clientStore.Disconnect()
 	clientStore.EnsureIndexes()
 
-	userapi := user.InitApi(config.User, logger, clientStore, highwater, marketoManager)
+	// Start logging kafka connection debug info
+	sarama.Logger = logger
+
+	kafkaConfig := events.NewConfig()
+	if err := kafkaConfig.LoadFromEnv(); err != nil {
+		log.Fatalln(err)
+	}
+	//I believe this should be removed at some point
+	kafkaConfig.SaramaConfig.Net.TLS.Config.InsecureSkipVerify = true
```

Comment:
This can be removed now - make sure you pull the latest version of `go-common`

Reply:
done